src/ml/neural_net/image_augmentation.hpp

```diff
@@ -308,7 +315,8 @@ class float_array_image_augmenter : public image_augmenter {
   };
 
   virtual float_array_result prepare_augmented_images(
-      labeled_float_image data_to_augment) = 0;
+      labeled_float_image data_to_augment, size_t output_height,
+      size_t output_width, bool resize_only) = 0;
```

Comment:
Why do these parameters need to be added? Isn't this captured in the `options` struct?